packages/create-cloudflare/src/cli.ts

```diff
@@ -117,78 +109,4 @@ const printBanner = () => {
 	startSection(`Create an application with Cloudflare`, "Step 1 of 3");
 };
 
-const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
```

Comment:
this shouldn't have been here, this function is a duplicate I introduced by mistake in #3807 after a rebase made before merging 🤦